src/utils/test/has-flags.spec.ts

```diff
@@ -0,0 +1,60 @@
+import { hasDebug, hasVerbose } from '@utils';
+
+describe('hasDebug', () => {
+  it('Returns true when a flag is passed', () => {
+    const config = {
+      flags: {
+        debug: true,
+        verbose: false,
+      },
+    };
+
+    expect(hasDebug(config)).toBe(true);
+  });
+
+  it('Returns false when a flag is not passed', () => {
+    const config = {
+      flags: {
+        debug: false,
+        verbose: false,
+      },
+    };
+
+    expect(hasDebug(config)).toBe(false);
+  });
+});
```

Comment:
To coincide with my test suggestion above:
```suggestion

  it('returns false when flags are not on the config', () => {
    const config = {};

    expect(hasDebug(config)).toBe(false);
  });
});
```